research/tax0/TAX0-ROUTER-FIX-SELECTION.md

```
@@ -0,0 +1,576 @@
+| cell | r1 | r2 | r3 |
+| --- | --- | --- | --- |
+| P0 D=8,C=8 | 0.993 (1.021) | 0.993 (1.002) | 1.218 (1.139) |
+| P0 D=8,C=32 | 0.666 (0.794) | 0.666 (0.777) | 0.817 (0.873) |
+| P0 D=8,C=128 | 0.287 (0.353) | 0.287 (0.346) | 0.352 (0.389) |
+| P0 D=8,C=512 | 0.088 (0.149) | 0.088 (0.145) | 0.108 (0.164) |
+| P0 D=32,C=32 | 0.994 (1.100) | 0.994 (1.002) | 0.917 (0.960) |
+| P0 D=32,C=128 | 0.359 (0.340) | 0.359 (0.310) | 0.325 (0.291) |
+| P0 D=32,C=512 | 0.101 (0.148) | 0.101 (0.134) | 0.092 (0.125) |
+| P0 D=128,C=128 | 0.997 (1.021) | 0.997 (1.010) | 0.400 (0.528) |
+| P0 D=128,C=512 | 0.231 (0.242) | 0.231 (0.238) | 0.093 (0.125) |
+| P1 (same 9 cells) | ≈P0 (max |Δ| 0.012) | ≈P0 | ≈P0 |
```

markdownlint-cli2 (0.23.2)

[warning] Table column count
Expected: 4; Actual: 6; Too many cells, extra data will be missing

(MD056, table-column-count)